homeassistant/components/binary_sensor/mychevy.py

```diff
@@ -0,0 +1,85 @@
+"""Support for MyChevy sensors.
+
+For more details about this platform, please refer to the documentation at
+https://home-assistant.io/components/mychevy/
+"""
+
+import asyncio
+from logging import getLogger
+
+from homeassistant.components.mychevy import (
+    EVBinarySensorConfig, DOMAIN, UPDATE_TOPIC
+)
+from homeassistant.components.binary_sensor import (
+    ENTITY_ID_FORMAT, BinarySensorDevice)
+from homeassistant.core import callback
+from homeassistant.util import (slugify)
+
+_LOGGER = getLogger(__name__)
```

Comment:
Please follow the same pattern as other modules.

Reply:
ack